Packs/OktaASA/Integrations/OktaASA/OktaASA_test.py

```diff
@@ -444,19 +446,30 @@ def test_search_events_limit_higher_then_1000_with_offset(mocker):
         OktaASAClient,
         "get_audit_events_request",
         side_effect=[
-            generate_results(False, datetime.now(), range_number_start=0, range_number_end=1000),
-            generate_results(
-                False,
-                datetime.now() + timedelta(seconds=1000),
-                range_number_start=1000,
-                range_number_end=2000,
-            ),
-            generate_results(
-                False,
-                datetime.now() + timedelta(seconds=2000),
-                range_number_start=2000,
-                range_number_end=2999,
-            ),
+            {
+                "list": generate_results(
+                    False, datetime.now(), range_number_start=0, range_number_end=1000
+                ),
+                "related_objects": {},
+            },
+            {
+                "list": generate_results(
+                    False,
+                    datetime.now() + timedelta(seconds=1000),
+                    range_number_start=1000,
+                    range_number_end=2000,
+                ),
+                "related_objects": {},
+            },
+            {
+                "list": generate_results(
+                    False,
+                    datetime.now() + timedelta(seconds=2000),
+                    range_number_start=2000,
+                    range_number_end=2999,
+                ),
+                "related_objects": {},
```

Comment:
Let's add a test when the related_objects is not empty (this is the point of this enhancement no?)